tests/foreman/cli/test_errata.py

```diff
@@ -60,16 +72,130 @@
+    VIRTUAL_MACHINES_COUNT = 2
+
+    @classmethod
+    @skip_if_not_set('clients', 'fake_manifest')
+    def setUpClass(cls):
+        """Create Org, Lifecycle Environment, Content View, Activation key,
+        Host, Host-Collection
+
+        """
+        super(HostCollectionErrataInstallTestCase, cls).setUpClass()
+        cls.org = make_org()
+        cls.env = make_lifecycle_environment({
+            'organization-id': cls.org['id']
+        })
+        cls.content_view = make_content_view({
+            'organization-id': cls.org['id']
+        })
+        cls.activation_key = make_activation_key({
+            'lifecycle-environment-id': cls.env['id'],
+            'organization-id': cls.org['id']
+        })
+        # add subscription to Satellite Tools repo to activation key
+        setup_org_for_a_rh_repo({
+            'product': PRDS['rhel'],
+            'repository-set': REPOSET['rhst7'],
+            'repository': REPOS['rhst7']['name'],
+            'organization-id': cls.org['id'],
+            'content-view-id': cls.content_view['id'],
+            'lifecycle-environment-id': cls.env['id'],
+            'activationkey-id': cls.activation_key['id']
+        })
+        # create custom repository and add subscription to activation key
+        setup_org_for_a_custom_repo({
+            'url': cls.CUSTOM_REPO_URL,
+            'organization-id': cls.org['id'],
+            'content-view-id': cls.content_view['id'],
+            'lifecycle-environment-id': cls.env['id'],
+            'activationkey-id': cls.activation_key['id']
+        })
+
+    def _create_host_collection(self):
+        """Create and setup host collection, hosts and virtual machines."""
+        self.virtual_machines = []
+        self.host_collection = make_host_collection({
+            'organization-id': self.org['id']
+        })
+        for _ in range(self.VIRTUAL_MACHINES_COUNT):
+            # create VM
+            virtual_machine = VirtualMachine(distro=DISTRO_RHEL7)
+            virtual_machine.create()
+            self.virtual_machines.append(virtual_machine)
+            virtual_machine.install_katello_ca()
+            # register content host
+            virtual_machine.register_contenthost(
+                self.org['name'],
+                activation_key=self.activation_key['name']
+            )
+            # enable red hat satellite repository
+            virtual_machine.enable_repo(REPOS['rhst7']['id'])
+            # install katello-agent
+            virtual_machine.install_katello_agent()
+            host = Host.info({'name': virtual_machine.hostname})
+            HostCollection.add_host({
+                'id': self.host_collection['id'],
+                'organization-id': self.org['id'],
+                'host-ids': host['id']
+            })
+            # because of the bz 1405434, install the custom package on each
+            # host here, rather than using HostCollection.package_install
+            virtual_machine.run(
```

Comment:
Thanks for comment describing why `run` is used here. However, to ensure we won't forget about BZ, i'd rather use `bz_bug_is_open` decorator and while BZ is open use `run`, as soon as BZ is closed - use `HostCollection.package_install`